SWLOR.Game.Server/Feature/MigrationDefinition/ServerMigration/_16_CorvetteRecipeAdditions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SWLOR.Game.Server.Core.Bioware;
+using SWLOR.Game.Server.Core.NWNX;
+using SWLOR.Game.Server.Core.NWScript.Enum.Item;
+using SWLOR.Game.Server.Entity;
+using SWLOR.Game.Server.Service;
+using SWLOR.Game.Server.Service.DBService;
+using SWLOR.Game.Server.Service.LogService;
+using SWLOR.Game.Server.Service.MigrationService;
+
+namespace SWLOR.Game.Server.Feature.MigrationDefinition.ServerMigration
+{
+    public class _16_CorvetteRecipeAdditions: ServerMigrationBase, IServerMigration
+    {
+        public int Version => 16;
+        public MigrationExecutionType ExecutionType => MigrationExecutionType.PostCacheLoad;
```

Comment:
Change this to MigrationExecutionType.PostDatabaseLoad.

PostDatabaseLoad = This happens after the database has loaded but before caching has started. You generally want to use this option because it's the earliest you can make changes to the database. The only time you should use PostCacheLoad is if you actually need something in the game cache (i.e recipes, starships, etc.)